src/python/bot/tasks/fuzz_task.py

```diff
@@ -1325,10 +1325,11 @@ def execute_task(fuzzer_name, job_type):
   if timeout_multiplier > 1:
     testcase_count /= timeout_multiplier
 
+  qemu_process = None
```

Comment:
nit: this line is unnecessary. variables set in the if have function scope.

Reply:
but we need access to qemu_process again on L1482, right?

Reply:
yeah, but it should still work, because it's only ever accessed when platform == 'FUCHSIA'